src/types/Wire.types.ts

```diff
@@ -17,86 +17,10 @@
  *
  */
 
+import {ClientConfig} from '../../server/config';
 import {App} from '../script/main/app';
 
-type FeatureVariables = {
```

Comment:
No more manual typing of this 😎  It's automatically inferred from the client config on the server side